### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by Azerrs** [Azerrs/PS4eop#2]
Reply to this comment if you got any questions about the emulator!

**Comment by ilan0098** [comment]
Is xmb menu included in the emulator?

**Comment by Azerrs** [comment]
> Is xmb menu included in the emulator?

Yes we are  making the full ps4 experience in this emulator so online party a store on our server to download a game

**Comment by ilan0098** [comment]
Sorry if i'm being a bit rude but where's the problem with the emulator?  I mean it sounds too good to be real , there must be something

**Comment by Azerrs** [comment]
> Sorry if i'm being a bit rude but where's the problem with the emulator? I mean it sounds too good to be real , there must be something

No don't worry about I understand can't wait to play the emulator, but there's is a lot of bugs the list is below.

Random Hard Crashes that crashes your os
Screen Flickering
Games not installing
Games not running
Saves Corrupting 
and a lot more umm so we just want to fix the bugs that make the emulator unusable, and we want to make the emulator more useable by the size the emulator itself is 21 gigabytes without any games installed so we are trying to code in a server for people that have low hard disk space they can have there games on our cloud that is a ten gigabit connection but thanks for asking

**Comment by ilan0098** [comment]
> > Sorry if i'm being a bit rude but where's the problem with the emulator? I mean it sounds too good to be real , there must be something
> 
> No don't worry about I understand can't wait to play the emulator, but there's is a lot of bugs the list is below.
> 
> Random Hard Crashes that crashes your os
> Screen Flickering
> Games not installing
> Games not running
> Saves Corrupting
> and a lot more umm so we just want to fix the bugs that make the emulator unusable, and we want to make the emulator more useable by the size the emulator itself is 21 gigabytes without any games installed so we are trying to code in a server for people that have low hard disk space they can have there games on our cloud that is a ten gigabit connection but thanks for asking

Thanks for the answer

**Comment by ilan0098** [comment]
The emulator will have online? PSN or your servers maybe?

**Comment by Azerrs** [comment]
> The emulator will have online? PSN or your servers maybe?

No PSN it's on our servers with a friends list and everything exactly like PSN
